### PR TITLE
Space team

### PR DESCRIPTION
**Pull request by luciagirasoles**

### CODE REVIEW

File: week-5-extended-project/team-space/db.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  country varchar (100)
+);
+
+CREATE TABLE agencies (
+  id SERIAL PRIMARY KEY,
+  name varchar (100),
+  contact_name varchar (100)
+);
+
+CREATE TYPE payment_type AS ENUM ('Visa','Mastercard', 'Amex', 'Diners');
```
Comment (CesarCachay) [inline week-5-extended-project/team-space/db.sql:22]: Could you explain me this please?

File: week-5-extended-project/team-space/seed.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+file_customers = File.open("seeds/customers.sql", "w")
+file_sales = File.open("seeds/sales.sql", "w")
+file_tours = File.open("seeds/tours.sql", "w")
+
+tours = []
+agencies = []
+customers = []
+
+data.each do |row|
+  row = row.map do |r|
```
Comment (sergiodxa) [inline week-5-extended-project/team-space/seed.rb]: What is `r`?

File: week-5-extended-project/team-space/db.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+CREATE TYPE pos_type AS ENUM ('Facebook','Whatsapp', 'Shop', 'Web');
```
Comment (MariethPerez) [inline week-5-extended-project/team-space/db.sql:23]: This is a good option!!!

Comment (stg101) [inline week-5-extended-project/team-space/db.sql:23]: I like the way you have created these types

File: week-5-extended-project/team-space/db.sql (hunk 1 above)
Comment (condef5) [reply week-5-extended-project/team-space/db.sql:22]: Enumerated (enum) types are data types that comprise a static, ordered set of values.
You can read more [here](https://www.postgresql.org/docs/10/datatype-enum.html)

File: week-5-extended-project/team-space/seed.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+require 'csv'
+
+headers, *data = CSV.read("data.csv")
```
Comment (MariethPerez) [inline week-5-extended-project/team-space/seed.rb:3]: Can you explain me please, what is the use of 'headers, *data'?

File: week-5-extended-project/team-space/seed.rb (hunk 2 above)
Comment (condef5) [reply week-5-extended-project/team-space/seed.rb]: Sorry, I just wanted to go through the array and add the single quote and I did not choose a good name for that variable.

File: week-5-extended-project/team-space/db.sql (hunk 3 above)
Comment (condef5) [reply week-5-extended-project/team-space/db.sql:23]: Thanks, you can read more [here](https://www.postgresql.org/docs/10/datatype-enum.html)

File: week-5-extended-project/team-space/queries.sql [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+-- Whatsapp is the channel where the organization must put all its efforts.
+SELECT s.pos, count(s.id), sum(t.price) as SALES from sales as s
+inner join tours as t on s.tour_id = t.id
+group by pos
+order by SALES desc;
+
+--Per each tour, where do customers come from? Show the top 5 locations
+select * from
+(select tmp.name, tmp.country, tmp.top, 
+ row_number() over (partition by tmp.name order by top desc) as rownum from
```
Comment (CesarCachay) [inline week-5-extended-project/team-space/queries.sql:37]: Using partitions is the only way that could answer this question without making 3 differents queries?

File: week-5-extended-project/team-space/seed.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  customers << row[8]
+  file_customers.puts "INSERT INTO customers VALUES (default, '#{row[7]}', '#{row[8]}', '#{row[9]}', '#{row[10]}');"
+  
+  unless agencies.include? row[5]
+    agencies << row[5]
+    file_agencies.puts "INSERT INTO agencies VALUES (default, '#{row[5]}', '#{row[6]}');"
+  end
+  
+  file_sales.puts "INSERT INTO sales VALUES (default, #{row[0]}, '#{row[4]}', '#{row[11]}', #{(tours.find_index row[1]) + 1}, #{(customers.find_index row[8]) + 1}, #{(agencies.find_index row[5]) + 1});"
+end
```
Comment (stg101) [inline week-5-extended-project/team-space/seed.rb:35]: Your code is readable and variable names are descriptive, great work.

File: week-5-extended-project/team-space/init.sh [hunk 7]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo "generate seeds"
+ruby seed.rb 
+echo "drop database"
+dropdb tours
+echo "create database"
+createdb tours
+echo "load tables in database"
+psql tours < db.sql
```
Comment (MariethPerez) [inline week-5-extended-project/team-space/init.sh:9]: Can you explain me please, how this line works?

File: week-5-extended-project/team-space/seed.rb (hunk 6 above)
Comment (condef5) [reply week-5-extended-project/team-space/seed.rb:35]: Thanks my friend

File: week-5-extended-project/team-space/queries.sql [hunk 8]
```diff
@@ -0,0 +1,40 @@
+--Show the total amount of sales in January 2009
+SELECT sum(t.price) from tours  as t 
+INNER JOIN sales as s on t.id = s.tour_id
+WHERE DATE_PART('month', s.date) = 1;
```
Comment (MariethPerez) [inline week-5-extended-project/team-space/queries.sql:4]: I would recomend adding a sentence to limit the year 2009.

**Pull request closed without merging**